### ISSUE TITLE
[Problem Report] File Sharing Over Local Network. The address is inaccessible and gives a timeout message.

### ISSUE THREAD

**Issue by LaZoRBear** [aleksey-hoffman/sigma-file-manager#22]
## System info:
- **App version**: 1.0.0
- **App page**: navigator
- **Operating System**: x64, win32, 10.0.19042
- **Free memory**: 6.2 GB
## Problem:
When sharing a file with local devices, the address is not accessible and throws a timeout.
- I have made sure that the app was allowed through the firewall
- I have made sure that my computer was visible on the network
- I have checked my router settings and nothing stood out

The app displays this error:
### Error:
```js
Error [ERR_IPC_CHANNEL_CLOSED]: Channel closed

    at ChildProcess.target.send (internal/child_process.js:705:16)

    at app://./index.bundle.js:1:37425

```
### App action history:
```js
09:23:16:961 | store.js::LOAD_DIR()
09:23:25:658 | store.js::LOAD_DIR()
09:23:28:557 | store.js::LOAD_DIR()
09:23:30:887 | store.js::LOAD_DIR()
09:23:31:939 | store.js::UPDATE_STORAGE_FILE()
09:23:31:945 | App.vue::initAllStorageFiles()
09:23:31:945 | store.js::LOAD_DIR()
09:23:32:205 | store.js::UPDATE_STORAGE_FILE()
09:40:03:219 | store.js::UPDATE_STORAGE_FILE()
09:40:03:227 | App.vue::initAllStorageFiles()
09:40:03:227 | store.js::LOAD_DIR()
09:40:03:486 | store.js::UPDATE_STORAGE_FILE()
09:40:05:283 | store.js::UPDATE_STORAGE_FILE()
09:40:05:287 | App.vue::initAllStorageFiles()
09:40:05:287 | store.js::LOAD_DIR()
09:40:05:544 | store.js::UPDATE_STORAGE_FILE()
09:49:49:536 | store.js::LOAD_DIR()
09:49:54:722 | store.js::LOAD_DIR()
09:50:12:815 | store.js::LOAD_DIR()
09:53:01:552 | store.js::LOAD_DIR()
09:53:02:014 | store.js::LOAD_DIR()
09:53:02:225 | store.js::LOAD_DIR()
09:53:02:372 | store.js::LOAD_DIR()
09:53:37:909 | store.js::LOAD_DIR()
09:54:15:586 | store.js::LOAD_DIR()
09:54:22:443 | store.js::UPDATE_STORAGE_FILE()
09:54:22:446 | App.vue::initAllStorageFiles()
09:54:22:446 | store.js::LOAD_DIR()
09:54:22:470 | App.vue::initAllStorageFiles()
09:54:22:470 | store.js::LOAD_DIR()
09:54:22:720 | store.js::UPDATE_STORAGE_FILE()
09:54:27:449 | store.js::LOAD_DIR()
09:54:29:260 | store.js::LOAD_DIR()
09:54:34:999 | store.js::LOAD_DIR()
09:54:42:235 | store.js::UPDATE_STORAGE_FILE()
09:54:42:236 | store.js::LOAD_DIR()
09:55:14:544 | store.js::LOAD_DIR()
09:55:14:713 | store.js::LOAD_DIR()
09:55:14:838 | store.js::LOAD_DIR()
09:55:14:987 | store.js::LOAD_DIR()
09:55:15:143 | store.js::LOAD_DIR()
09:55:16:028 | store.js::UPDATE_STORAGE_FILE()
09:55:16:037 | App.vue::initAllStorageFiles()
09:55:16:037 | store.js::LOAD_DIR()
09:55:16:287 | store.js::UPDATE_STORAGE_FILE()
09:55:27:017 | store.js::LOAD_DIR()
13:04:54:657 | store.js::LOAD_DIR()
13:04:56:279 | store.js::LOAD_DIR()
13:05:59:476 | store.js::LOAD_DIR()
13:06:01:170 | store.js::LOAD_DIR()

```

**Comment by aleksey-hoffman** [comment]
Thank you for reporting this issue. I've seen it before. It seems like it's happening after switching from `stream` to `download` type.

**Comment by aleksey-hoffman** [comment]
Fixed in 99f5792
Will be released in v1.4 (on Saturday)
